packages/cashier/src/components/cashier-locked-checker/cashier-locked-checker.tsx

```diff
@@ -21,7 +24,7 @@ const CashierLockedChecker: React.FC<React.PropsWithChildren<unknown>> = observe
         );
 
     if (is_system_maintenance) {
-        if (is_cashier_locked || (is_deposit_locked && currency_config.is_crypto)) {
+        if (is_cashier_locked || ((is_deposit_locked || is_withdrawal_locked) && currency_config.is_crypto)) {
```

Comment:
Add the `is_withdrawal_locked` check to the generic `CashierLockedChecker` component.